features/environment.py

```diff
@@ -0,0 +1,34 @@
+import os
+from behaving import environment as benv
+
+PERSONAS = {
+    'Test_user': dict(
+            username=os.environ['SANITYUSER'],
+            password=os.environ['SANITYPASS'],
+            Atmosphere="https://atmo.iplantcollaborative.org/application"
```

Comment:
I think it might be useful to make this an environment variable as well, so we can choose which environment to test. Make it default to production, like so:

```python
atmosphere=os.environ.get('SANITYSITE', 'https://atmo.iplantcollaborative.org/application')
```